database/index.js

```diff
@@ -5,8 +5,19 @@ const connection = mysql.createConnection({
   database: 'yelpest'
 });
 
-
+getBusinessPhotos = (busId, callback) => {
```

Comment:
Shouldn't this have a variable declaration in front to prevent making a global variable?
const getBusinessPhotos = ....